### ISSUE TITLE
[1.18.2-5.2-hf] Riding Trigger doesn't work

### ISSUE THREAD

**Issue by Huetete** [TheComputerizer/Music-Triggers#28]
Korporal K. Reep#0531 and Manam#0802 reported that riding trigger crashes the moment they mount a mob. Song block example used:

```toml
[horse]
	[horse.trigger]
		name = "riding"
		priority = "201"
		resource_name = "minecraft:horse;minecraft:donkey;minecraft:mule;minecraft:boat;minecraft:strider"
		identifier = "ride"
```

Crash Report from Korporal K. Reep#0531:

https://pastebin.com/n3NzYJcq

**Comment by TheComputerizer** [comment]
Probably fixed in 5.3, but I can re open this if its still an issue

**Comment by DRACTION** [comment]
Still crash on 5.3
```
[blue_bird_naruto]
	[blue_bird_naruto.trigger]
		name = "riding"
		resource_name = "minecraft:pig"
		identifier = "pig"
```
